portfolio/src/main/webapp/index.html

```diff
@@ -4,46 +4,177 @@
 <head>
 	<meta charset="UTF-8">
 	<title>My Portfolio</title>
+	<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
+		integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
 	<link rel="stylesheet" href="style.css">
+	<script src="https://code.jquery.com/jquery-3.4.1.slim.min.js"
+		integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n" crossorigin="anonymous">
+	</script>
+	<script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
+		integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous">
+	</script>
+	<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"
+		integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous">
+	</script>
 	<script src="script.js"></script>
 </head>
 
 <body>
 	<div id="content">
-		<h1>Claire's Portfolio</h1>
+        <br><br>
+		<h1>Claire's Portfolio  
+             <a href=" https://github.com/apluscs ">
```

Comment:
Looks like you've got mixed tabs and spaces here. It probably looks right in your editor, but I think github uses different defaults, so its weird here.

My preference would be to use spaces consistently, so something like a find-and-replace for the tab character and replace it with your preferred number of spaces.

Reply:
Marking resolved

(feel free to do this yourself in the future once you've resolved a comment)